lib/shelley/test/data/cardano-node/node.cert

```diff
@@ -0,0 +1,8 @@
+{ "omega": 0
+, "issuerPk":
+    "KbEP1cIgT+ITFB/+isxQYT+yN0T47WHTZ43gvgy8kROJt9/Lsf25kpxRNU/AFsXeCnARfnwfIf2pCkTcaO1z6Q=="
+, "delegatePk":
+    "b8j8qcnO/HyB5PcDgROOrF3fGHyAe1Ec2vitqVas+ZfWdrcaTfRDIhcKT+6jsNh0oQuKi7I23shW+1X7V3Y2Pw=="
+, "cert":
+    "ea983bfd989be8d1ac1529e0d9f9ef723766397ad58ea24b89d09c40fc3122a8728f3efaca35f16ab994aa02fff76098006889f20810c40d47a34a3a6b9d3e05"
+}
```

Comment:
I'd assume that this only applies to Byron. Shelley uses VRF/KES credentials for stake pools.